webapp/src/filelist.tsx

```diff
@@ -308,7 +308,8 @@ class FileTreeItem extends sui.StatelessUIElement<FileTreeItemProps> {
     }
 
     renderCore() {
-        const { onClick, onItemClick, onItemRemove, isActive, hasDelete, file, meta, ...rest } = this.props;
+        // onErrorClick is explicitly set so that it is not passed down under the ...rest
+        const { onClick, onItemClick, onItemRemove, isActive, hasDelete, file, meta, onErrorClick, ...rest } = this.props;
```

Comment:
Doesn't look like onErrorClick is being used

Reply:
It isn't used here, it's used in the ``handleErrorClick`` method above, this is pulling it out of the ``...rest`` to prevent it from propagating to the anchor element; is there a better way to structure here? I just fixed this one real quick to stop getting errors, nothing came to mind immediately